src/pages/cart/cart.jsx

```diff
@@ -1,5 +1,60 @@
-export function Cart(){
-    return(
-        <h1>This is Cart page</h1>
-    )
-}
\ No newline at end of file
+import { useCartWishlist, useProduct, useAuth } from "contexts";
+import { HorizontalProductCard } from "components";
+import { EmptyPageMessage } from "../empty-page-message/empty-page-message";
+import { useNavigate, Navigate } from "react-router-dom";
+import { useEffect } from "react";
+
+export function Cart() {
+  document.title = "My Cart | Crickstock";
+  const { cartItems, cartTotalEstimate } = useCartWishlist();
+  const { price, discount, totalPrice, totalCartItems } = cartTotalEstimate;
+
+  return cartItems[0] ? (
```

Comment:
just curious why this condition for cartItems[0] and not size?

Reply:
I just preferred this here because I only need to check whether the array is empty, so if its first index is empty it'll be empty, so I don't need to use the `.length` property which will iterate over the whole array.